### DISCUSSION TITLE
Saleor 3.0 beta :  How to deploy dashboard in Apache server ?

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#8876] [Deployment]
![dashGraph](https://user-images.githubusercontent.com/56902009/147551096-735aee8b-5b1a-4726-a453-65016aca8a3a.PNG)
![InkeddashGraph2_LI](https://user-images.githubusercontent.com/56902009/147551739-7ec3f24f-00d6-488e-9494-df96cd0c242f.jpg)
 i couldn't solve this problem while long time. Help me   @karolkielecki , @maarcingebala , @orzechdev

**Comment by karolkielecki** [comment]
@Batgerel53 Hi. Why would you like to deploy the dashboard on Apache? Saleor Dashboard is a static app--it doesn't require an additional server. You can deploy it using Docker or, e.g., Amazon S3.
